modules/QnA/resources/qnaService.js

```diff
@@ -164,7 +164,10 @@
             return this.cuePoint().text;
         };
 
-        this.isAnswerOnAirActive = function(playerTime) {
+        this.isAnswerOnAirActive = function(playerTime,player) {
+            if(player && player.embedPlayer.isDVR()){
```

Comment:
@OrenMe 
We are talking about live+dvr 
player.embedPlayer.currentTime is holding a relative time **in seconds** and not absolute timestamp. Is this a bug? Should it be holding the absolute time? (and if we switch this - wouldn't it break many other components?)